core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -614,6 +614,67 @@ public void hour() {
     assertEquals("hour(\"2020-08-17 01:02:03\")", expression.toString());
   }
 
+  public void testHourOfDay(FunctionExpression dateExpression, int hour) {
```

Comment:
Is this function used anywhere else? If not, I'd make it `private`?

Reply:
Fixed in efe6258331e277a121ae522069c717295d802dde